scania-digital-design-system/sdds
# feat(card): migrate card component

**Describe pull-request**  
Migration of the card component

**Solving issue**  
Fixes: [AB#2384](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2384)

**How to test**  
1. Go to storybook link below
2. Check in components -> card
3. Check that the design reflects the one in Figma

**Screenshots**  
-

**Additional context**  
-
